MIGRATION.md

```diff
@@ -21,14 +25,14 @@ We separate only icons from the main package. If you need to use our icons, plea
 
 When using NPM
 
-```
+```bash
```

Comment:
maybe we need to do `static=true` here?

Reply:
Not really, this is only used on the github repo, not on the website

Reply:
my spellchecker was complaining about code block without a type, I'll update it to sh